app/app.go

```diff
@@ -421,30 +421,31 @@ func NewStrideApp(
 		appCodec, keys[ibchost.StoreKey], app.GetSubspace(ibchost.ModuleName), app.StakingKeeper, app.UpgradeKeeper, scopedIBCKeeper,
 	)
 
-	// Create RateLimit Keeper
+	// Create Ratelimit Keeper
 	scopedratelimitKeeper := app.CapabilityKeeper.ScopeToModule(ratelimitmoduletypes.ModuleName)
 	app.ScopedratelimitKeeper = scopedratelimitKeeper
 	app.RatelimitKeeper = *ratelimitmodulekeeper.NewKeeper(
 		appCodec,
 		keys[ratelimitmoduletypes.StoreKey],
 		app.GetSubspace(ratelimitmoduletypes.ModuleName),
 		app.BankKeeper,
+		app.IBCKeeper.ChannelKeeper,
 	)
 	ratelimitModule := ratelimitmodule.NewAppModule(appCodec, app.RatelimitKeeper)
 
 	// Create ICS4Wrapper middleware
-	rateLimitingParams := app.GetSubspace(ratelimitmoduletypes.ModuleName)
-	rateLimitingParams = rateLimitingParams.WithKeyTable(ratelimitmoduletypes.ParamKeyTable())
-	rateLimitingICS4Wrapper := ratelimitmodule.NewICS4Middleware(
+	rateLimitParams := app.GetSubspace(ratelimitmoduletypes.ModuleName)
+	rateLimitParams = rateLimitParams.WithKeyTable(ratelimitmoduletypes.ParamKeyTable())
```

Comment:
I think we don't need these two lines anymore as we don't use rateLimitParams anymore in app.go

Reply:
Good catch! 

Do we still need the params stuff everywhere else? (e.g. params.proto, keeper/params.go, types/params.go)?